gulpfile.js

```diff
@@ -177,6 +183,20 @@ pipes.buildArtifacts = function () {
 
 // == TASKS ========
 
+gulp.task('bump', function() {
+    var pkg = getPackageJson();
+
+    var newVer = pkg.version;
+
+    gulp.src('./bower.json')
+    .pipe(bump({version: newVer}))
+    .pipe(gulp.dest('./'));
+
+    gulp.src('./app/modules/version.json')
+    .pipe(bump({version: newVer + '-' + process.env.BUILD_NUMBER}))
```

Comment:
Jenkins should automatically set this environment variable, so gulp should be able to pick it up when Jenkins hits the gulp task.